frb_example/with_flutter/lib/bridge_generated.dart

```diff
@@ -8,6 +8,7 @@ import 'dart:async';
 import 'package:meta/meta.dart';
 import 'package:flutter_rust_bridge/flutter_rust_bridge.dart';
 import 'package:uuid/uuid.dart';
+import 'ffi.io.dart' if (dart.library.html) 'ffi.web.dart';
```

Comment:
Main issue: I am not sure, but seems that the `late final api = ...` can happen *anywhere* in the user code, instead of hardcoded into this file.

Thus, maybe we can do something like this:

1. Add a command line argument like `--extra-header-dart`. The implementation is simple - just copy whatever user provides into the top of these generated dart files.
2. `--extra-header-dart "import 'ffi.io.dart' if (dart.library.html) 'ffi.web.dart';"` for this specific frb example.
3. No more need to hardcode importing which file.

Reply:
I think convention is better than configuration.
If it must be, it could do another pr.

Reply:
Hmm, But this forces everyone to use ffi.io.dart and ffi.web.dart to store the api instance, which I am worried whether it is too much constraint. Take Spring Boot for example - it prefers conventions over configuration, but at the same time, it does allow configuration, so anyone who needs customization can do it.